src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/DocFilesHandlerImpl.java

```diff
@@ -198,7 +198,6 @@ private void handleHtmlFile(DocFile srcfile, DocPath dstPath) throws DocFileIOEx
         PackageElement pkg = dfElement.getPackageElement();
 
         HtmlDocletWriter docletWriter = new DocFileWriter(configuration, dfilePath, element, pkg);
-        configuration.messages.notice("doclet.Generating_0", docletWriter.filename.getPath());
```

Comment:
I guess this is to prevent duplicate output. It could probably have been solved by printing the HTML file's whole path (`docletWriter.filename` just contains the base name), or including the destination path. But I'm not convinced we need to generate a message for each and every file we copy, so it's probably fine as it is.

Reply:
Please ignore that comment, I just saw there's already a message with the full path.

Reply:
FWIW, based on other recent experiences, I think there is the potential to have more versatile control of which messages to suppress. `-quiet` just does everything. But this is another discussion.